freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/StackToCloudStackConverter.java

```diff
@@ -244,6 +276,20 @@ private InstanceAuthentication buildInstanceAuthentication(StackAuthentication s
                 stackAuthentication.getLoginUserName());
     }
 
+    private GroupNetwork buildGroupNetwork(com.sequenceiq.freeipa.entity.Network network) {
```

Comment:
Note: As discussed, this logic will have to be adjusted to fetch the subnet ID list from the `InstanceGroup` (freeipa) in a later PR. Nothing to do now.